bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -137,16 +139,22 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
             reply.println("This pull request will have to be integrated manually using the " +
                     "[/integrate](https://wiki.openjdk.org/display/SKARA/Pull+Request+Commands#PullRequestCommands-/integrate) pull request command.");
             return;
-        } else if (commandArg == Command.defer) {
-            pr.addLabel("deferred");
-            reply.println("Integration of this pull request has been deferred and may be completed by any project committer using the " +
+        } else if (commandArg == Command.defer || commandArg == Command.delegate) {
+            pr.addLabel("delegated");
+            if (commandArg == Command.defer) {
+                reply.println("FutureWarning: /integrate defer is deprecated and will be moved in a future version. Use /integrate delegate instead.");
```

Comment:
```suggestion
                reply.println("Warning: /integrate defer is deprecated and will be removed in a future version. Use `/integrate delegate` instead.");
```

Reply:
I will also make /integrate defer to ` /integrate defer`